src/events-header.tsx

```diff
@@ -50,6 +51,12 @@ export default function EventsHeader({ user,
 }: EventsHeaderProps) {
     const navigate = useNavigate();
 
+    const handleTripleClick = useLongPress( () =>  {
+        if (onLogoTripleClicked) {
+            onLogoTripleClicked();
```

Comment:
long press should be double click not triple - i.e. get to the settings

Reply:
Fixed